setup.py

```diff
@@ -1,11 +1,26 @@
+import io
+import os
+import re
+
 from setuptools import setup, find_packages
 
+
+def find_version():
+    file_dir = os.path.dirname(__file__)
+    with io.open(os.path.join(file_dir, "incognia", "__init__.py")) as f:
+        version = re.search(r'^__version__ = [\'"]([^\'"]*)[\'"]', f.read())
+        if version:
+            return version.group(1)
+        else:
+            raise RuntimeError("Unable to find version string.")
```

Comment:
Is this safe enough? What happens when we fall into the error case?

Reply:
It was suggested [here](https://packaging.python.org/en/latest/guides/single-sourcing-package-version/);
But, this change was made in another PR and should be merged before this one:
 - https://github.com/inloco/incognia-python/pull/15;